tests/basic_env.py

```diff
@@ -98,6 +100,23 @@ def setUp(self):
 
     def tearDown(self):
         self._popd()
+        try:
+            shutil.rmtree(self._root_dir)
+        except OSError:
+            # We ignore this under Windows with a warning because it happened
+            # to be really hard to trace all not properly closed files.
+            #
+            # Best guess so far is that gitpython is the culprit:
+            # it opens files and uses __del__ to close them, which can happen
+            # late in current pythons. TestGitFixture and TestDvcFixture try
+            # to close that and it works on most of the tests, but not all.
+            # Repos and thus git repos are created all over the dvc ;)
+            if os.name == "nt":
+                warnings.warn(
+                    "Failed to remove test dir, some file is still opened"
```

Comment:
Let's at least include str for the exc so that it is easier to debug in the future. Btw, is there some specific errno that we need to ignore or is it all over the place?